packages/ibm-products/src/components/InlineTip/utils.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const getComponentText = (reactNode) => {
```

Comment:
Is this similar to the `getNodeTextContent` utility you created? If so, how is this different?

Reply:
_(This was originally Jeff's code, and I haven't tested this feature extensively.)_ InlineTip's `getComponentText()` is designed to take all the contents of the `children`/content passed to it, of any size and format, reduced to a single block of text, for the purpose of displaying a single line of truncated text when the inline tip is "collapsed".

 _(What I wrote:)_ `getNodeTextContent()` was designed to reduce a short amount of text such as `<p>Title <b>Page</b>.</p>` to a plain string `"Title Page."`. This is ideal for `aria-labels`, which would render `<p>Title <b>Page</b>.</p>` as `[Object object]`. It's also been adapted elsewhere for `aria-region`, checkbox labels, and datagrid row headers.

It's possible Jeff's function is more robust, but I haven't tested it.